static/app/components/tables/simpleTable/index.tsx

```diff
@@ -79,13 +78,20 @@ function RowCell({
   children,
   className,
   justify,
-}: {
+  ...props
+}: ComponentProps<typeof Flex> & {
```

Comment:
I think ComponentProps<typeof Flex> should be all we need here? since it has justify, classname, children